client/landing/gutenboarding/path.ts

```diff
@@ -43,6 +43,10 @@ export const path = [ '', ...Object.values( routeFragments ) ].join( '/' );
 export type StepType = ValuesType< typeof Step >;
 export type StepNameType = keyof typeof Step;
 
+export interface GutenLocationStateType {
```

Comment:
Should we add a comment here or change the name to indicate that this type holds Anchor.fm information? Or should it stay more generic?

Reply:
This name, I think is appropriate, since whenever location state is used in Gutenboarding, it must conform to this type.